addenda13_test.go

```diff
@@ -78,14 +78,10 @@ func TestMockAddenda13(t *testing.T) {
 func testAddenda13ValidRecordType(t testing.TB) {
 	addenda13 := mockAddenda13()
 	addenda13.recordType = "63"
-	if err := addenda13.Validate(); err != nil {
-		if e, ok := err.(*FieldError); ok {
-			if e.FieldName != "recordType" {
-				t.Errorf("%T: %s", err, err)
-			}
-		} else {
-			t.Errorf("%T: %s", err, err)
-		}
+	err := addenda13.Validate()
+	// TODO: are we not expecting any errors here?
```

Comment:
return the fieldError	

if addenda13.recordType != "7" {
		fieldError("recordType", NewErrRecordType(7), addenda13.recordType)
	}